ZupIT/beagle-web-react
# Fix/retry beagle error

**- What I did**
Following the pattern of the component on the Android environment, I created the same behavior on the component, allowing the user to retry the action that he was trying to do. Also, I added the errors list, that is provided by the core, showing to the user what could happened.

P.S.: The detailed messages are hidden and only will be displayed if the user clicks on "Show details".

**- How I did it**
Followed the pattern of components already developed on the project, keeping the behavior of Android environment and added useful information about what happened, giving details about the errors.

**- How to verify it**
1. After building the project, link it using yarn and create a new React project installing beagle. 
2. Then link your built project
3. On your test React project, create a JSON with this content:
`{
            "_beagleComponent_": "beagle:lazycomponent",
            "path": "/listview.json",
            "initialState": {
                "_beagleComponent_": "beagle:text",
                "text": "Loading the screen, please wait",
                "textColor": "grey",
                "alignment": "CENTER"
            }
        }`
4.  Then start your application, it should display the error message, because we do not have created the "listview.json" that _beagle:lazycomponent_ requires, showing you the error component.

**- Description for the changelog**
